app/sidekiq/kms_key_rotation/rotate_keys_job.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module KmsKeyRotation
+  class RotateKeysJob
+    include Sidekiq::Worker
+
+    sidekiq_options retry: 5, queue: :low
+
+    def perform(args)
+      gids = args['gids'] || args[:gids] # rspec didn't like args['gids']
+      Rails.logger.info { "Re-encrypting records: #{gids.join ', '}" }
+      records = GlobalID::Locator.locate_many gids
+
+      skip_hq_callback do
+        records.each do |r|
+          r.rotate_kms_key!
+        rescue => e
```

Comment:
☝️

Reply:
@ericboehs so I did this one so that we'd have the ID of the record throwing the error. That could be useful, I think